src/microbe_stage/MicrobeHUD.cs

```diff
@@ -807,7 +807,20 @@ private void UpdateHealth(float delta)
 
     private void UpdatePopulation()
     {
-        populationLabel.Text = stage.GameWorld.PlayerSpecies.Population.ToString(CultureInfo.InvariantCulture);
+        var playerPopulation = stage.GameWorld.PlayerSpecies.Population;
+
+        if (playerPopulation < 1000)
```

Comment:
There's already a static helper method for the formatting: `StringUtils.FormatNumber`. I'd use it here since it also handles translations.